### PR TITLE
Changes from LAMA test deployment on 2020 05 09

### PR DESCRIPTION
**Pull request by padthaitofuhot**
We mostly touched Docker stuff.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -38,3 +38,14 @@ yarn-debug.log*
 .env.test.local
 .env.local
 .env.localsendgrid.env
+
+# ignore IDEA environment files.
```
Comment (maebeale) [inline .gitignore:42]: why not .idea/* ? 
:)

File: README.md [hunk 2]
```diff
@@ -125,14 +125,22 @@ To get started using the application with docker,
 2. Install [docker-compose](https://docs.docker.com/compose/install/)
 3. Clone the repository, and open the repository folder in your favorite command line or terminal application.
 4. From within the repository, navigate to the `/docker/development` folder. If you are in the right folder, you will see a file named `docker-compose.yml`.
-5. Run the `rake secret` command. If the `rake` command is not found, you may need to run `bundle install` from the repository's root folder. Once the `bundle install` command has completed successfully, return to the `docker/development` folder and try `rake secret` again.
```
Comment (maebeale) [inline README.md:128]: was rake secret not working?

File: config/environments/development.rb [hunk 3]
```diff
@@ -37,7 +37,7 @@
   config.action_mailer.perform_caching = false
 
   # per devise's suggested settings
-  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  config.action_mailer.default_url_options = { host: , port: 3000 }
```
Comment (maebeale) [inline config/environments/development.rb:40]: the comma stays here?

**Comment by padthaitofuhot** [comment]
Right, rake only exists in the container.

**Comment by padthaitofuhot** [comment]
Because auto-injectors.  If it matters I can try to change it or just
remove the commit.  Or expand on it and add the .vscode directory to it,
too. ;)

**Comment by maebeale** [comment]
Oh, it's fine! Just curious.

Def would like to take the vs code folder out.

**Comment by maebeale** [comment]
Merging now so we can test

**Pull request merged**